cost-analyzer/values.yaml

```diff
@@ -198,6 +198,8 @@ kubecostFrontend:
     #limits:
     #  cpu: "100m"
     #  memory: "256Mi"
+#  customerGoogleTag allows you to embed your Google Global Site Tag to track usage of Kubecost.
```

Comment:
@teevans sorry for catching this late, but what if we moved this under the reporting block? that's where all related analytics flags are. 

https://github.com/kubecost/cost-analyzer-helm-chart/blob/master/cost-analyzer/values.yaml#L609